libnetwork/internal/nftables/nftables_linux.go

```diff
@@ -147,7 +147,15 @@ func Enabled() bool {
 	return nftPath != ""
 }
 
-// ////////////////////////////
+// Disable undoes Enable. Intended for unit testing.
+func Disable() {
```

Comment:
I wish there was a way to share `_test.go` field between tests. I guess some `if testing.Testing()` would come closest to prevent it being used for other purposes, but of course would still compile the code.

Well, "meh" it's an internal package, so should not be an issue.

Reply:
Yeah, not worth changing; if we'd use the `test.Testing()` approach, we'd indeed need some error return to make it not ignored silently

Reply:
Heh, yeah, I considered `panic`, but then thought "not worth it for an internal package".